vmaas/tests/conftest.py

```diff
@@ -15,11 +16,45 @@ def pytest_configure(config):
     config.addinivalue_line('markers', 'smoke: mark a test as a smoke test.')
 
 
+@pytest.fixture(name='make_conn')
+def _make_conn():
+    def make_conn():
+        hostname = conf.get('hostname', 'localhost')
+        try:
+            hostname, port = hostname.split(':')
+        except ValueError:
+            port = 8080 if hostname in ('localhost', '127.0.0.1') else 80
+        return VMaaSClient(hostname, port=port)
+
+    yield make_conn
+
+
 @pytest.fixture()
-def rest_api():
+def rest_api(make_conn):
+    return make_conn()
+
+
+@pytest.fixture(scope="session", autouse=True)
+def cache_bash():
+    """
+    Each process of webapp has its own cache. Cache responses for bash
+    package update request, with and without filters.
+    Helps to catch https://github.com/RedHatInsights/vmaas/issues/306
+    """
+
     hostname = conf.get('hostname', 'localhost')
     try:
         hostname, port = hostname.split(':')
     except ValueError:
         port = 8080 if hostname in ('localhost', '127.0.0.1') else 80
-    return VMaaSClient(hostname, port=port)
+    rest_api = VMaaSClient(hostname, port=port)
```

Comment:
Why not using the ``make_conn`` fixture?

Reply:
Because I wrote this before I had `make_conn` fixture :smile: and forget to change this. Thanks!